pkg/config/types/publishers.go

```diff
@@ -40,7 +44,7 @@ type S3Publisher struct {
 	PreSignedURLExpiration Duration `yaml:"PreSignedURLExpiration,omitempty"`
 }
 
-func (c S3Publisher) Installed() bool {
+func (c S3Publisher) IsConfigured() bool {
 	return c != S3Publisher{}
 }
```

Comment:
similar issue here. s3 publisher and input rely on credentials deployed on the node and can be enabled even with empty config